microsoft/vscode
# Update package.json

Fixes #146093



This PR fixes #146093

Visual Studio Code will now correctly identify the user's operating system and show an accurate message on Linux.

To test:
1. Build Visual Studio Code.
2. Install Git and Visual Studio Code in a Debian virtual machine.
3. Initialize a new Git repository.
4. Uninstall Git.
5. Open the SCM sidebar and verify that the message is correct.
